[2022-09-13 20:16:39,497] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.test_python scheduled__2022-09-12T00:00:00+00:00 [queued]>
[2022-09-13 20:16:39,524] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.test_python scheduled__2022-09-12T00:00:00+00:00 [queued]>
[2022-09-13 20:16:39,525] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:39,527] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-13 20:16:39,529] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:39,583] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): test_python> on 2022-09-12 00:00:00+00:00
[2022-09-13 20:16:39,615] {standard_task_runner.py:52} INFO - Started process 208 to run task
[2022-09-13 20:16:39,649] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'test_python', 'scheduled__2022-09-12T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/pre_parser.py', '--cfg-path', '/tmp/tmpme6paovf', '--error-file', '/tmp/tmprir4wjii']
[2022-09-13 20:16:39,652] {standard_task_runner.py:80} INFO - Job 13: Subtask test_python
[2022-09-13 20:16:39,883] {task_command.py:369} INFO - Running <TaskInstance: test_dag.test_python scheduled__2022-09-12T00:00:00+00:00 [running]> on host 5d92941617fc
[2022-09-13 20:16:40,085] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=test_python
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T00:00:00+00:00
[2022-09-13 20:16:40,089] {logging_mixin.py:115} INFO - {}
[2022-09-13 20:16:40,090] {python.py:173} INFO - Done. Returned value was: None
[2022-09-13 20:16:40,151] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=test_python, execution_date=20220912T000000, start_date=20220913T201639, end_date=20220913T201640
[2022-09-13 20:16:40,231] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-13 20:16:40,335] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
